cypress.json

```diff
@@ -0,0 +1,6 @@
+{
+  "baseUrl": "http://localhost:9000",
+  "video": false,
```

Comment:
Did we ever figure out if we were storing screenshots/screencaptures of the tests run? I think this option turns off the auto capture of the cypress tests

Reply:
Yes, I swithed off the video capture. We decided only to take a screenshot after every test passed (I'm gonna add this in future prs). Of course it's not problem to switch video back